async fn test() -> Box<dyn std::error::Error> {
    macro!()
}

fn main() {}
